### PR TITLE
Testing cleanups for PR 2731

### PR DESCRIPTION
**Pull request by semenko**
Tiny nits for PR #2731

### CODE REVIEW

**Comment by semenko** [comment]
@TheNavigat Mind taking a look?

File: test/chromium/script.py [hunk 1]
```diff
@@ -15,52 +15,62 @@
...
+    sys.exit(1)
 
-print '' #New line
+print ''
 
 driver.quit()
+
+
+if test_failed:
+    sys.exit(1)
```
Comment (TheNavigat) [inline test/chromium/script.py]: This line is unreachable.

**Comment by semenko** [comment]
Good call! Anything else?

**Comment by TheNavigat** [comment]
Nope, LGTM. Good work!

**Pull request merged**